website/src/controllers/api-v3/groups.js

```diff
@@ -109,7 +109,9 @@ api.getGroups = {
           }).select(groupFields).sort(sort).exec()); // TODO use lean?
           break;
         case 'tavern':
-          queries.push(Group.getGroup({user, groupId: 'habitrpg', fields: groupFields, populateLeader: true}));
+          if (types.indexOf('publicGuilds') === -1) {
```

Comment:
Eh, sorry about that. I didn't notice it was a parameter. I was thinking it was just in the projection.